JS/modules/dateTime.js

```diff
@@ -0,0 +1,30 @@
+import { DateTime } from "./luxon.js";
+
+const dateLine = document.querySelector(".dayDate");
+const navBar = document.querySelector("nav");
+/* eslint-disable-next-line */
+export const displayDate = () => {
+  const dt = DateTime.now();
+  const f = { month: "long", day: "numeric" };
+  dt.setLocale("en-US").toLocaleString(f);
+  dateLine.innerHTML = `${dt}`;
+  navBar.appendChild(dateLine);
+};
+
+// var dt = DateTime.now();
+// var f = {month: 'long', day: 'numeric'};
+// dt.setLocale('fr').toLocaleString(f)      //=> '14 septembre'
+// dt.setLocale('en-GB').toLocaleString(f)   //=> '14 September'
+// dt.setLocale('en-US').toLocaleString(f)
+
+// export let displayDate = () => {
+//   const stringDate = new Date();
+//   const date = stringDate.toLocaleDateString('en-us', {
+//     year: 'numeric',
+//     month: 'long',
+//     day: 'numeric',
+//   });
+//   const time = stringDate.toLocaleTimeString();
+//   dateLine.innerHTML = `${date} ${time}`;
+//   navBar.appendChild(dateLine);
+// };
```

Comment:
- [ ] I would like to **request** you kindly `remove` the comments from your codebase because `commented` code adds unnecessary noise to version control systems like **Git**. It can make it harder to track `meaningful` changes in the *codebase* over time. **Version** control systems are designed to track code history, and having commented-out code in the **repository** can create confusion and lead to `unnecessary` *conflicts* during code merging or branching.